### PR TITLE
Updates for adding commercial Azul Java bits

### PR DESCRIPTION
**Pull request by shrishrirang**
- 8.5-jre8-alpine and 9.0-jre8-alpine directories are unchanged.
- 8.5.-jre8 and 9.0-jre8 now use appsvc/java as the base image, which contains the commercial Azul Java bits

### CODE REVIEW

**Comment by shrishrirang** [comment]
@jvano @vinisoto

File: setup.sh [hunk 1]
```diff
@@ -9,3 +9,7 @@ cp -r shared 8.5-jre8-alpine/tmp/shared
 rm -rf 9.0-jre8-alpine/tmp/shared
 mkdir -p 9.0-jre8-alpine/tmp
 cp -r shared 9.0-jre8-alpine/tmp/shared
```
Comment (jvano) [inline setup.sh]: Please remove the apline folders since those won't be supported

File: 9.0-jre8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
+ARG BASE_IMAGE=appsvc/java:8-jre8_0000000000
+FROM $BASE_IMAGE
+MAINTAINER Azure App Services Container Images <appsvc-images@microsoft.com>
+
+ENV AI_VERSION 2.1.0
```
Comment (jvano) [inline 9.0-jre8/Dockerfile]: >2.1.0 [](start = 15, length = 5)

Please make sure we update AI to the latest version

**Comment by jvano** [comment]
@vinisoto is added to the review. #Closed

File: 8.5-jre8/Dockerfile [hunk 3]
```diff
@@ -24,18 +15,38 @@ ENV JAVA_OPTS -Dsite.home=/home $JAVA_OPTS
...
+COPY background.png /tmp/webapps/ROOT/background.png
+COPY sshd_config /etc/ssh/
+COPY tmp/shared/app_insights/AI-Agent.xml /usr/local/app_insights/aiagent/
+COPY tmp/shared/app_insights/ApplicationInsights.xml /usr/local/app_insights/tomcat_lib/
 
 RUN apt-get update \
-        && apt-get install -y --no-install-recommends dialog \
-        && apt-get update \
-        && apt-get install -y --no-install-recommends openssh-server \
+        && apt-get install -y --no-install-recommends dialog openssh-server wget \
```
Comment (vinisoto) [inline 8.5-jre8/Dockerfile]: I think openssh-server is already retrieved in the base image

File: setup.sh (hunk 1 above)
Comment (shrishrirang) [reply setup.sh]: Keeping it for a while for reference, if needed.

File: 8.5-jre8/Dockerfile (hunk 3 above)
Comment (shrishrirang) [reply 8.5-jre8/Dockerfile]: Right! removed.

File: 9.0-jre8/Dockerfile (hunk 2 above)
Comment (shrishrirang) [reply 9.0-jre8/Dockerfile]: Done!

**Pull request merged**